landscaper/pkg/controlplane/blueprint/blueprint.yaml

```diff
@@ -74,12 +74,54 @@ imports:
   - name: runtimeCluster
     required: true
     targetType: landscaper.gardener.cloud/kubernetes-cluster
+    type: target
   - name: virtualGarden
     required: false
     schema:
       ref: blueprint://schema/com.github.gardener.gardener.landscaper.pkg.controlplane.apis.imports.v1alpha1.VirtualGarden.yaml
   
 
+exports:
+  
```

Comment:
```suggestion
```

Reply:
That's a generated blueprint, I had troubles with it because otherwise the generated code would end up on the same line as the `exports:`. 
I do not think it matters much as no human will probably read this (just for the landscaper as validation).

Reply:
The following diff should do the trick:
```diff
diff --git a/landscaper/pkg/controlplane/generate/templates/blueprint-controlplane.tpl.yaml diff --git a/landscaper/pkg/controlplane/generate/templates/blueprint-controlplane.tpl.yaml
b/landscaper/pkg/controlplane/generate/templates/blueprint-controlplane.tpl.yaml
index 9b68b337b..d764844d1 100644
--- a/landscaper/pkg/controlplane/generate/templates/blueprint-controlplane.tpl.yaml
+++ b/landscaper/pkg/controlplane/generate/templates/blueprint-controlplane.tpl.yaml
@@ -5,10 +5,10 @@ kind: Blueprint
 # For reference, see: https://json-schema.org/specification-links.html#draft-4
 jsonSchema: "http://json-schema.org/draft-04/schema"
 imports:
-  {{ .imports | nindent 2 }}
+{{ .imports | indent 2 }}

 exports:
-  {{ .exports | nindent 2 }}
+{{ .exports | indent 2 }}

 # For what can be referenced in the deployExecutions, please see:
 # https://github.com/gardener/landscaper/blob/master/docs/usage/Blueprints.md#deployexecuti
ons
```

Reply:
I have done that but the new line is still there.

Reply:
I suggest moving ahead as this is a minor detail

Reply:
Hm, I actually testing it when I suggested it and the new line was gone for me.